frontend/src/views/login/entrance/index.vue

```diff
@@ -42,10 +33,7 @@
 
 <script setup lang="ts" name="login">
 import LoginForm from '../components/login-form.vue';
-import UnSafe from '@/components/error-message/unsafe.vue';
-import ErrIP from '@/components/error-message/err_ip.vue';
 import ErrCode from '@/components/error-message/error_code.vue';
-import ErrDomain from '@/components/error-message/err_domain.vue';
 import ErrFound from '@/components/error-message/404.vue';
 import { ref, onMounted } from 'vue';
 import { GlobalStore } from '@/store';
```

Comment:
There doesn't seem to be any apparent errors in the current implementation of Vue components you provided. However, it's important to note that modern web development practices often recommend using TypeScript and componentization for better organization. 

I would suggest reviewing these files with the latest updates from the TypeScript community:

1. Use `const` instead of `var`, which is more efficient when declared multiple times.
2. Avoid inline styles if possible because they can cause side effects (e.g., updating elements' content without refreshing) and should generally use class or computed properties where applicable.

Since the code was checked on September 2021, some changes might have occurred since then, including additional features or bug fixes not addressed here. Ensure to review them against up-to-date versions of Vue.js and dependencies used.
It's also good practice to test these components through automated tests to make sure things still work properly after any modifications are made.